syntax = "proto3";

option csharp_namespace = "Autobarn.PricingEngine";

package price;

// The greeting service definition.
service Pricer {
  // Sends a greeting
  rpc GetPrice (PriceRequest) returns (PriceReply);
}

// The request message containing the user's name.
message PriceRequest {
  string manufacturer = 1;
  string model = 2;
  string color = 3;
  sint32 year = 4;
}

// The response message containing the greetings.
message PriceReply {
  sint32 price = 1;
  string currencyCode = 2;
}
